### PR TITLE
MAUI-3102 Addressed the review changes in Autocomplete and ComboBox control UG

### PR DESCRIPTION
**Pull request by anandhan-rajagopal**
Addressed the review changes in Autocomplete and ComboBox control:
 * Changed IsDropDownOpen content in Autocomplete.
 * Changed Migration content in Autocomplete and ComboBox control.

### CODE REVIEW

**Comment by anandhan-rajagopal** [comment]
rebuild

File: MAUI/Autocomplete/Migration.md [hunk 1]
```diff
@@ -53,4 +49,52 @@ To migrate easier from Xamarin SfAutoComplete to .NET MAUI `SfAutocomplete`, we
...
-</table> 
\ No newline at end of file
+<tr>
+<td>SuggestionMode <br/> AutoCompleteMode</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the search mode of the control. The enum values are "StartsWith" and "Contains". <br/>When using StartsWith, the AutoCompleteMode will be set to Append. <br/> When using Contains, the AutoCompleteMode will be set to Suggest.</td></tr>
+<tr>
+<td>ItemsSource,<br/>DataSource,<br/>ComboBoxSource. </td>
+<td>ItemsSource</td>
+<td>Gets or sets the items source to be populated in the control.</td></tr>
```
Comment (PaulAndersonS) [inline MAUI/Autocomplete/Migration.md:59]: ItemsSource is a single word. change as Gets or sets the data to be populated

File: MAUI/Autocomplete/Selection.md [hunk 2]
```diff
@@ -193,26 +193,26 @@ The following image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete](Images/Selection/IsClearButtonVisible.png)
 
-## Open a drop-down programmatically
+## Is Drop-Down Open
 
-In `Autocomplete` control, the drop-down can be opened or closed programmatically by using the `IsDropDownOpen` property. The default value of `IsDropDownOpen` property is `false`. 
+The "IsDropDownOpen" property in the "Autocomplete" control can be used to get a boolean value indicating whether the drop-down is open or closed. It returns `true` if the drop-down is open, otherwise `false`.
```
Comment (PaulAndersonS) [inline MAUI/Autocomplete/Selection.md:198]: whether the drop down is in open or closed state

File: maui-toc.html [hunk 3]
```diff
@@ -477,6 +477,6 @@
...
....2.0.36">v20.2.0.36 Main release</a></li></ul></li><li>2022 volume 1 - v20.1.0.*<ul><li> Weekly Nuget Release <ul><li><a href="/maui/release-notes/v20.1.0.56">v20.1.0.56</a></li><li><a href="/maui/release-notes/v20.1.0.52">v20.1.0.52</a></li><li><a href="/maui/release-notes/v20.1.0.51">v20.1.0.51</a></li><li><a href="/maui/release-notes/v20.1.0.50">v20.1.0.50</a></li><li><a href="/maui/release-notes/v20.1.0.48">v20.1.0.48</a></li></ul></li><li><a href="/maui/release-notes/v20.1.0.55">v20.1.0.55 Service Pack Release</a></li><li><a href="/maui/release-notes/v20.1.0.47">v20.1.0.47 Main Release</a></li></ul></li><li>2021 Volume 4  - v19.4.0.*<ul><li><a href="/maui/release-notes/v19.4.0.48">v19.4.0.48 Service Pack Release</a></li><li><a href="/maui/release-notes/v19.4.0.38">v19.4.0.38 Main Release</a></li></ul></li><li>2021 volume 3 - v19.3.0.*<ul><li> Weekly Nuget Release <ul><li><a href="/maui/release-notes/v19.3.0.45">v19.3.0.45</a></li><li><a href="/maui/release-notes/v19.3.0.44">v19.3.0.44</a></li></ul></li><li><a href="/maui/release-notes/v19.3.0.53">v19.3.0.53 Service Pack Release</a></li><li><a href="/maui/release-notes/v19.3.0.43">v19.3.0.43 Main Release</a></li></ul></li></ul>
```
Comment (PaulAndersonS) [inline maui-toc.html:480]: Is this change needed

File: MAUI/Release-notes/v20.3.0.47.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Essential Studio for  MAUI 2022 volume 3 Main Release Notes  
+description: Essential Studio for  MAUI 2022 volume 3 Main Release Notes 
+platform: maui
+documentation: ug
+---
+
+# Essential Studio for  MAUI Release Notes  
+
```
Comment (PaulAndersonS) [inline MAUI/Release-notes/v20.3.0.47.md:9]: Is this change needed. Rebase your branch with development

File: MAUI/Autocomplete/Selection.md (hunk 2 above)
Comment (anandhan-rajagopal) [reply MAUI/Autocomplete/Selection.md]: Changes are made

File: MAUI/Autocomplete/Migration.md (hunk 1 above)
Comment (anandhan-rajagopal) [reply MAUI/Autocomplete/Migration.md]: Changes are made

**Review by PaulAndersonS — APPROVED** [review]
Fine
